app/_src/gateway/upgrade-v2.md

```diff
@@ -80,6 +79,29 @@ The following table outlines various upgrade path scenarios to {{page.kong_versi
 
 {% endif_version %}
 
+{% if_version eq: 3.3.x %}
```

Comment:
@outsinre For 3.3, is each row correct? I wasn't sure if any other direct upgrades were possible in 3.3

Reply:
LGTM. But I suggest that we add more reviewers.